.gitlab-ci.yml

```diff
@@ -137,23 +137,36 @@ convert_raspbian_raspberrypi4:
   variables:
     RASPBERRYPI_PLATFORM: raspberrypi4
 
+.bash_helpers: &bash_helpers |
+  # Bash function to check if the string is a final tag
+  function is_final_tag () {
+    version="$1"
+    [[ "${version}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]] && return 0 || return $?
+  }
+  # Bash function to check if the string is a build tag
+  function is_build_tag () {
+    version="$1"
+    [[ "${version}" =~ ^[0-9]+\.[0-9]+\.[0-9]+-build[0-9]+$ ]] && return 0 || return $?
+  }
+
 .template_test_acceptance: &test_acceptance
   stage: test_acceptance
   needs:
     - job: build
       artifacts: true
-  image: ubuntu:22.04
+  image: ubuntu:jammy
```

Comment:
* I would appreciate a comment that `jammy == 22.04`. I don't know all those names by heart! :older_man:

* Since `jammy` is repeated quite a lot below, could we make it a variable instead?

Reply:
I think making it a variable (and adding a comment) is a good idea, I'll follow up :+1:

The reason why I changed to the codename is that it used below, so that whenever updating it here you would see the references below. However having a variable makes it even easier!